src/cares_wrap.cc

```diff
@@ -1266,8 +1266,14 @@ static void Initialize(Handle<Object> target,
               Integer::New(env->isolate(), AF_UNSPEC));
   target->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "AI_ADDRCONFIG"),
               Integer::New(env->isolate(), AI_ADDRCONFIG));
+#ifdef __FreeBSD__
```

Comment:
Maybe #ifdef on AI_V4MAPPED?  It's probably an issue on DragonFlyBSD as well, maybe the other BSDs too.